packages/stark-core/src/modules/http/services/http.service.spec.ts

```diff
@@ -1534,8 +1534,8 @@ function convertMapIntoObject(map: Map<string, any>): { [param: string]: any } {
 class HttpServiceHelper<P extends StarkResource> extends StarkHttpServiceImpl<P> {
 	public retryDelay: number;
 
-	public constructor(logger: StarkLoggingService, sessionService: StarkSessionService, $http: HttpClient) {
-		super(logger, sessionService, $http);
+	public constructor(logger: StarkLoggingService, sessionService: StarkSessionService, $http: SpyObj<HttpClient>) {
```

Comment:
Maybe could we change `$http` by `httpClient` ? 😊 

Because `$http` is linked to AngularJS and not anymore to Angular.
`$http` mention only appears in this spec file.

Reply:
True. I'll fix it 👍